client/app/src/accounts/account.controller.js

```diff
@@ -120,7 +120,9 @@
     ]
 
     // 1 ARK has 100000000 "arkthosi"
-    const UNIT = Math.pow(10, 8)
+    const ARKTOSHI_VAL = Math.pow(10, 8)
```

Comment:
I think ARKTOSHI_UNIT may be a better name for it. Also because you've renamed UNIT, you'll need to do the same for all other UNIT occurrences in this file. For example:

[Line 516](https://github.com/ckhatri/ark-desktop/blob/ad209b9cfda83322f3c45c6cbca2e3a9e6309f95/client/app/src/accounts/account.controller.js#L516)
[Line 657](https://github.com/ckhatri/ark-desktop/blob/ad209b9cfda83322f3c45c6cbca2e3a9e6309f95/client/app/src/accounts/account.controller.js#L657)

Reply:
Thanks for the catch! Whoops.

Reply:
No worries! We've all been there 😆

Reply:
Yup! Got 'em all I think.

Reply:
Hmm. Sorry, it may not work still. For example, the first snippet below is looking for a global variable / a const called "ARKTOSHI_UNIT", but it's currently called ARKTOSHI_VAL.

```
amount: parseInt(resp.send_amount * ARKTOSHI_UNIT)
```

You could either rename the const to ARKTOSHI_UNIT or remove the const completely and just reference self.ARKTOSHI_UNIT.

```
const ARKTOSHI_UNIT = Math.pow(10, 8)
```

or

```
amount: parseInt(resp.send_amount * self.ARKTOSHI_UNIT)
```

Does that make sense?

Reply:
If I'm wrong and it works, please say! I've not tested yet (I tend to check code first then test the end result) 😄